AEPIdentity/Sources/IdentityHitProcessor.swift

```diff
@@ -44,6 +47,15 @@ class IdentityHitProcessor: HitProcessing {
         }
     }
 
+    func shouldQueue(entity: DataEntity) -> Bool {
+        // only queue hits if Edge is not registered
+        let shouldQueue = isEdgeRegistered()
+        if !shouldQueue {
+            Log.debug(label: "\(LOG_TAG):shouldQueue", "Dropping hit with id: '\(entity.uniqueIdentifier)' because Edge is present.")
```

Comment:
This log is a bit misleading as the hit is just not queued in Identity anymore, but the data is still going to be processed. We should log a message at the beginning of the processing saying that `AEP Edge is enabled, sending identifiers in XDM format` or something similar.